src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.SingleCheckout/VueComponents/CheckoutStepVueComponent.ts

```diff
@@ -69,7 +69,7 @@ module Orckestra.Composer {
                             isLastStep: this.$parent.isLastStep,
                             id: this.id,
                             active: this.active,
-                            displayContinueButton: (!this.checked && (this.nextStepId() === this.id)),
+                            displayContinueButton: (((this.checked && !this.fulfilled) || !this.checked) && (this.nextStepId() === this.id)),
```

Comment:
((this.checked && !this.fulfilled) || !this.checked) can be simplify to  (!this.checked || !this.fulfilled)